src/pages/text/team-chat-api/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+weight: 80
+category: "text"
+title: Overview
+tagline: "Build messaging software with Team Chat API by Text."
+desc: "API for a two-way communication between users. Available as a Web API."
+---
+
+# Introduction
+
+Team Chat API by Text is a backend service that allows for a two-way communication between users using the HTTP protocol (Web reference). It supports the communication between agents/team members, not between customers and agents.
+
+Team Chat API won't let you customize the LiveChat application, but it will let you build your own chat application from scratch.
+
+## Features
+
+- Communication can happen on public channels or via direct messages
+- Storing messaging history
+- Users can see messages that were sent before they joined the channel
+
+## Web reference
+
+The Web Reference section contains information about the Team Chat API's RESTful API and is available in the OpenAPI format.
+
+<SectionLink to={"/text/team-chat-api/web-reference"}>
+  See Web Reference
+</SectionLink>
+
+## FAQ
+
+### Is Team Chat API the same as Agent Chat API?
+
+No. Team Chat API is a separate service independent of the LiveChat product.
+
+### Can I use Team Chat API to chat with customers?
+
+Team Chat API was designed for building custom chat applications for internal communication (agent - agent; team member - team member).
+
+### Can I build apps for LiveChat with Team Chat API?
+
+It's not designed for building apps for LiveChat. If you're looking for a solution to chat with customers, you should use the [Agent Chat API](https://developers.livechat.com/docs/messaging/agent-chat-api/) and [Customer Chat API](https://developers.livechat.com/docs/messaging/customer-chat-api/).
```

Comment:
Actually, this is not 100% true. This API is designed to be independent from LiveChat, but an app for LiveChat can definitely use this API.

Reply:
You're right. I meant something else, I need to rephrase it.